examples/DDPG/train.py

```diff
@@ -0,0 +1,100 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import gym
+import numpy as np
+import time
+from mujoco_agent import MujocoAgent
+from mujoco_model import MujocoModel
+from parl.algorithms import DDPG
+from parl.utils import logger
+from replay_memory import ReplayMemory
+from ou_strategy import OUStrategy
+
+MAX_EPISODES = 5000
+MAX_STEPS_EACH_EPISODE = 1000
+ACTOR_LR = 1e-4
+CRITIC_LR = 1e-3
+GAMMA = 0.99
+TAU = 0.001
+MEMORY_SIZE = int(1e6)
+MIN_LEARN_SIZE = 1e4
+BATCH_SIZE = 128
+REWARD_SCALE = 1.0
+ENV_SEED = 1
+
+
+def main():
```

Comment:
Could you have a common structure for each `train.py` among different directories? 
It would be easier for users to read our codes.